models/models.go

```diff
@@ -94,7 +95,22 @@ func Setup(c *config.Config) error {
 		return err
 	}
 	// Open our database connection
-	db, err = gorm.Open(conf.DBName, conf.DBPath)
+	max_attempts := 10
+	i := 0
+	for{
```

Comment:
Please run `gofmt` on your code to ensure that formatting is kept consistent across the project. Something that's helped me a ton is to set my editor to automatically run `goimports` (basically `gofmt` + import management) every time the file is saved.

Reply:
ran `gofmt` on the last push